source/FreeRTOS_DNS_Cache.c

```diff
@@ -122,13 +163,16 @@
  * @post the global structure \a xDNSCache might be modified
  */
     BaseType_t FreeRTOS_dns_update( const char * pcName,
-                                    uint32_t * pulIP,
-                                    uint32_t ulTTL )
+                                    IPv46_Address_t * pxIP,
+                                    uint32_t ulTTL,
+                                    BaseType_t xLookUp,
```

Comment:
xLookUp is added based on IPv6 code base. https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/labs/ipv6_multi/source/FreeRTOS_DNS_Cache.c#L186

addressinfo is added because in some cases we are passing addressinfo.